### ISSUE TITLE
TCL CANT UPDATE

### ISSUE THREAD

**Issue by XAI-Pro** [neutronscott/flip2#19]
Hi!

First of all, thanks so much for your great work its really amazing and I'm really enjoying it!!

So I ended up following your steps and rooting my TCL before I updated it and after I realized that there a few issues with the TCL in the old software (the calendar is not showing properly etc.) so I tried to update the phone but I get a message "your phone is rooted and cant be updated"

So I was wondering if you can please help me out and instruct me how to update my phone?
(by either updating the phone still while in root or by unrooting the phone and then updating and then after following your instructions again to allow app installations 

thank you!!

(and don't worry I will still be able to allow USB debugging thought an app that I have even with the new TCL software)

**Comment by technobulb** [comment]
Yes the fact that they shipped a phone with a broken calendar (& more) is pretty sad...
What is the exact phone model and software version that you are using?

**Comment by XAI-Pro** [comment]
Thank you so much for responding!
I’m using the tcl flip 2 T408DL build KEE7

**Comment by TheProgrammer26** [comment]
I had the same issue. Did you @XAI-Pro find a solution?

**Comment by neutronscott** [comment]
the neutron.img is based of KEEZ OTA update. The stock one is here: https://scottn.us/downloads/phone3/
The rest there are KEE7!
I just stocked my KEE7 and got KEEZ OTA, but it gives no updates afterwards... You?

**Comment by TheProgrammer26** [comment]
So is there a way to fix the calender?

**Comment by neutronscott** [comment]
You can apply the stock boot.img, allow to update to KEEZ, and then re-apply rooted neutron.img to boot.
If you applied any edits to partitions you'd want to reflash those as well. I have just done this on my KEE7 and it updated to KEEZ and re-rooted OK.

**Comment by anonymousfliphones** [comment]
try flashing an updated stock super for example a KEKA version

**Comment by flipphone2** [comment]
> the neutron.img is based of KEEZ OTA update. The stock one is here: https://scottn.us/downloads/phone3/ The rest there are KEE7! I just stocked my KEE7 and got KEEZ OTA, but it gives no updates afterwards... You?

do you still have the KEEZ stock boot? link broken. thanks
